main.py

```diff
@@ -503,14 +503,16 @@ def handle_new_switch(self, event):
         """Create a new Device on the Topology.
 
         Handle the event of a new created switch and update the topology with
-        this new device.
+        this new device. Also notify if the switch is enabled.
         """
         switch = event.content['switch']
         switch.activate()
         log.debug('Switch %s added to the Topology.', switch.id)
         self.notify_topology_update()
         self.update_instance_metadata(switch)
         self.restore_network_status(switch)
+        if switch.is_enabled:
```

Comment:
```suggestion
        if switch.is_enabled():
```
`switch.is_enabled` is a method.

Reply:
Solved in https://github.com/kytos/topology/pull/155/commits/4793919422a9f513404272ba82c1c9749466bb25, a new call was also added to store administrative changes when Kytos is running with `-E` in https://github.com/kytos/topology/pull/155/commits/d314458cf6115d59065909756980717c2ccf1c63